testapi.pm

```diff
@@ -78,7 +78,6 @@ our $username;
 our $password;
 
 our $serialdev;
-our $selected_console;
```

Comment:
For such a change, we need to bump isotovideo interface number at least https://github.com/os-autoinst/os-autoinst/blob/master/isotovideo#L66

Reply:
Indeed

Reply:
@alvarocarvajald, @ldevulder, @kravciak

Reply:
Yes we use it with @alvarocarvajald in our tests. Can we keep this function? Or is there another way to do this?

Reply:
@ldevulder We're trying to come up with a middle ground ^^

Reply:
After this change, it will be under autotest rather than testapi. But we probably should have an official api call to provide this.

Reply:
Oups sorry! I should have read all the code before asking dumb question :)

Reply:
Yeah, I'm using `$testapi::selected_console` in all tests below `sles4sap` :(

Reply:
@okurz @alvarocarvajald I am wondering if it would be better to implement a function similar to `is_serial_terminal` instead of checking for the exact console using this variable. I don't think tests should be accessing this variable directly anyway, it is an implementation detail (hence why I had to move it to change the implementation).

We should at least create a function in test API to wrap it, but I wonder if you really need to know the exact console and not just whether the console has a particular property?

Reply:
@richiejp I just needed to know the currently selected console to be able to get back to it in my tests

Reply:
@richiejp https://github.com/os-autoinst/os-autoinst/pull/992